jest.setup.ts

```diff
@@ -42,6 +40,7 @@ beforeEach(() => {
   global.FRONTEND_DOMAIN = 'frontend.serlo.org'
   global.FRONTEND_PROBABILITY = '1'
   global.FRONTEND_ALLOWED_TYPES = '[]'
+  global.fetch = jest.fn().mockImplementation(fetch)
```

Comment:
```suggestion
  global.fetch = jest.fn().mockImplementation(fetchNode)
```

A slidely better version :-)